docs/command_line_tutorial.md

```diff
@@ -0,0 +1,90 @@
+# Command-Line Tutorial
+
+This tutorial will explain how to use the HELM command line tools to run benchmarks, aggregate statistics, and visualize results.
+
+In this tutorial, we will run two runs using the `mmlu` scenario on the `huggingface/gpt-2` model. The `mmlu` scenario implements the **Massive Multitask Language (MMLU)** benchmark from [this paper](https://arxiv.org/pdf/2009.03300.pdf), and consists of a Question Answering (QA) task using a dataset with questions from 57 subjects such as elementary mathematics, US history, computer science, law, and more. We will run two runs: the first using questions about anatomy, and the second using questions about philosophy.
```

Comment:
Should say that GPT-2 will work horribly on MMLU, so this is just a proof of concept.